src/integration-test/java/apptest/DocumentTypeIT.java

```diff
@@ -0,0 +1,104 @@
+package apptest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.LOCATION;
+import static org.springframework.http.HttpMethod.DELETE;
+import static org.springframework.http.HttpMethod.GET;
+import static org.springframework.http.HttpMethod.PATCH;
+import static org.springframework.http.HttpMethod.POST;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+import static org.springframework.http.HttpStatus.OK;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.jdbc.Sql;
+
+import se.sundsvall.dept44.test.AbstractAppTest;
+import se.sundsvall.dept44.test.annotation.wiremock.WireMockAppTestSuite;
+import se.sundsvall.document.Application;
+import se.sundsvall.document.integration.db.DocumentTypeRepository;
+
+@WireMockAppTestSuite(files = "classpath:/DocumentTypeIT/", classes = Application.class)
+@Sql({
+	"/db/scripts/truncate.sql",
+	"/db/scripts/testdata-it.sql"
+})
+class DocumentTypeIT extends AbstractAppTest {
+
+	private static final String PATH = "/%s/admin/documenttypes";
+	private static final String REQUEST_FILE = "request.json";
+	private static final String RESPONSE_FILE = "response.json";
+
+	@Autowired
+	private DocumentTypeRepository repository;
+
+	@Test
+	void test01_createDocumentType() {
```

Comment:
It would be nice with a separate GET call afterwards to the path given in the location header to also ensure that the location header is correct.